packages/gestalt-datepicker/src/DateRange.tsx

```diff
@@ -358,16 +358,11 @@ function DateRange({
                     onSecondaryDateChange({ value: startDate }, { value: endDate });
                   }
                 }}
-                rangeEndDate={
-                  selectedRange === DateRangeType.Primary
-                    ? dateValue.endDate
-                    : secondaryDateValue?.endDate
-                }
-                rangeStartDate={
-                  selectedRange === DateRangeType.Primary
```

Comment:
Why does this move to the Internal component now?

Reply:
For this same reason: https://github.com/pinterest/gestalt/pull/3910#discussion_r1882938621